samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -42,14 +42,15 @@
 
     <!-- Images -->
     <MauiImage Include="Resources\Images\*" />
+    <MauiImage Update="Resources\Images\spash.svg" BaseSize="128,128" Color="#512BD4" Resize="false" />
```

Comment:
1. Typo splash.svg
2. shouldn't it be applied to MauiSplashScreen?

Reply:
@VladislavAntonyuk  Please see the "Additional Information" above and the other comments.  This is specifically a problem with the Windows platform, where the splash screen icon is not found when using just MauiSpashScreen.

Reply:
@VladislavAntonyuk I not the typo and will fix.